domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/service/searchcriteria/SearchCriteriaService.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.ccd.definition.store.domain.service.searchcriteria;
+
+import uk.gov.hmcts.ccd.definition.store.repository.entity.SearchCriteriaEntity;
+import uk.gov.hmcts.ccd.definition.store.repository.model.SearchCriteria;
+
+import java.util.List;
+
+public interface SearchCriteriaService {
+
+    void saveAll(List<SearchCriteriaEntity> entityList);
+
+    List<SearchCriteria> findByCaseTypeReferences(List<String> caseTypeReferences);
```

Comment:
where do we use this method? `findByCaseTypeReferences` ?

Reply:
If not used can you remove from SearchPartyService as well?

Reply:
done in [59c375d1](https://github.com/hmcts/ccd-definition-store-api/pull/1061/commits/59c375d172d81b375f2fe43039955b022afd906e)